examples/games/game_menu.rs

```diff
@@ -369,8 +410,8 @@ mod menu {
     ) {
         for (interaction, button_setting, entity) in &interaction_query {
             if *interaction == Interaction::Pressed && *setting != *button_setting {
-                let (previous_button, mut previous_image) = selected_query.single_mut();
-                previous_image.color = NORMAL_BUTTON;
+                let (previous_button, mut previous_color) = selected_query.single_mut();
```

Comment:
This should stay as `previous_image` as well.

Reply:
Same as above, also reverted in next commit.